packages/nx/src/plugins/js/project-graph/build-dependencies/explicit-project-dependencies.ts

```diff
@@ -56,34 +56,29 @@ export function buildExplicitTypeScriptDependencies(
     '.mjs',
     '.cjs',
     '.cts',
+    '.vue',
   ];
 
-  // TODO: This can be removed when vue is stable
-  if (isVuePluginInstalled()) {
-    moduleExtensions.push('.vue');
-  }
-
-  for (const [project, fileData] of Object.entries(
-    ctx.filesToProcess.projectFileMap
-  )) {
+  Object.keys(ctx.filesToProcess.projectFileMap).forEach((project) => {
```

Comment:
Does this actually yield a performance benefit?

Reply:
Yes, I don't have the numbers anymore, but it shaved off 100+ms IIRC.